include/hitwriter.h

```diff
@@ -15,6 +15,8 @@ struct HitCounterInfo {
   uint64_t NumHits;
 
   virtual void setPath(const std::string&) {}
+
+  virtual void setBuffer(const char*, size_t, uint64_t) {}
```

Comment:
It's not clear what this interface is for or what the arguments mean.

Reply:
Can you add the parameter names to the interface and one-line comment about what this is for?